app/helpers/loss_report_helper.rb

```diff
@@ -0,0 +1,189 @@
+module LossReportHelper
+
+  def cmr_client
+    Cmr::Client.client_for_environment(Rails.configuration.cmr_env, Rails.configuration.services)
+  end
+
+  def token
+    if session[:login_method] == 'launchpad'
+      session[:launchpad_cookie]
+    elsif session[:login_method] == 'urs'
+      session[:access_token]
+    end
+  end
+
+  def loss_report_output(hide_items=true, disp='text')
```

Comment:
I have sat with this for a while, and the usage of all the instance variables feels wrong to me. I can't cite a specific rails or ruby best practice but here are a few reasons:
- as this is a helper file (and not in the controller), using instance variables directly in the methods makes this confusing. I think normally you would call helper methods in the view file and pass in the instance variables as parameters
- as it is (maybe this will change but based on my understand of how the endpoint will work I don't think it will) this isn't actually being used by a view, so this might be better in a different file, in which case the instance variables might not be available (I am thinking of a concern or service object, but that is not something you need to worry about for this PR)
- along with the previous point of this potentially moving elsewhere in the future, if you want to use these methods in the rake task (which I think we should seriously consider as to not have duplicate code), the instance variables would not work and that information would need to be passed in

Bottom line, I think you should be passing in all the method arguments instead of using instance variables here.